### PR TITLE
Fixes ?. nonsense.

### PR DESCRIPTION
**Pull request by AnturK**
Latest 514 revealed these.

### CODE REVIEW

File: code/game/objects/items/toy_mechs.dm [hunk 1]
```diff
@@ -446,19 +446,23 @@
 /obj/item/toy/prize/proc/check_battle_start(mob/living/carbon/user, obj/item/toy/prize/attacker, mob/living/carbon/target)
 	if(attacker?.in_combat)
 		to_chat(user, "<span class='notice'>[target?target.p_their() : "Your" ] [attacker.name] is in combat.</span>")
-		target?.to_chat(target, "<span class='notice'>Your [attacker.name] is in combat.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>Your [attacker.name] is in combat.</span>")
```
Comment (Fikou) [inline code/game/objects/items/toy_mechs.dm:450]:
```suggestion
		to_chat(target, "<span class='notice'>Your [attacker.name] is in combat.</span>")
```

File: code/game/objects/items/toy_mechs.dm [hunk 2]
```diff
@@ -446,19 +446,23 @@
...
 		return FALSE
 	if(in_combat)
 		to_chat(user, "<span class='notice'>Your [name] is in combat.</span>")
-		target?.to_chat(target, "<span class='notice'>[target.p_their()] [name] is in combat.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>[target.p_their()] [name] is in combat.</span>")
```
Comment (Fikou) [inline code/game/objects/items/toy_mechs.dm:455]:
```suggestion
		to_chat(target, "<span class='notice'>[user.p_their()] [name] is in combat.</span>")
```
the fact that it uses your for the user but target's their for the target seems unintended?

File: code/game/objects/items/toy_mechs.dm [hunk 3]
```diff
@@ -446,19 +446,23 @@
...
 		to_chat(user, "<span class='notice'>[target?target.p_their() : "Your" ] [attacker.name] isn't ready for battle.</span>")
-		target?.to_chat(target, "<span class='notice'>Your [attacker.name] isn't ready for battle.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>Your [attacker.name] isn't ready for battle.</span>")
 		return FALSE
 	if(timer > world.time)
 		to_chat(user, "<span class='notice'>Your [name] isn't ready for battle.</span>")
-		target?.to_chat(target, "<span class='notice'>[target.p_their()] [name] isn't ready for battle.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>[target.p_their()] [name] isn't ready for battle.</span>")
```
Comment (Fikou) [inline code/game/objects/items/toy_mechs.dm:465]:
```suggestion
		to_chat(target, "<span class='notice'>[user.p_their()] [name] isn't ready for battle.</span>")
```

File: code/game/objects/items/toy_mechs.dm [hunk 4]
```diff
@@ -446,19 +446,23 @@
...
 		to_chat(user, "<span class='notice'>Your [name] is in combat.</span>")
-		target?.to_chat(target, "<span class='notice'>[target.p_their()] [name] is in combat.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>[target.p_their()] [name] is in combat.</span>")
 		return FALSE
 	if(attacker && attacker.timer > world.time)
 		to_chat(user, "<span class='notice'>[target?target.p_their() : "Your" ] [attacker.name] isn't ready for battle.</span>")
-		target?.to_chat(target, "<span class='notice'>Your [attacker.name] isn't ready for battle.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>Your [attacker.name] isn't ready for battle.</span>")
```
Comment (Fikou) [inline code/game/objects/items/toy_mechs.dm:460]:
```suggestion
		to_chat(target, "<span class='notice'>Your [attacker.name] isn't ready for battle.</span>")
```
```suggestion
		to_chat(target, "<span class='notice'>Your [attacker.name] isn't ready for battle.</span>")
```

**Review by Fikou — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
Lol who merged this

**Comment by Mothblocks** [comment]
image

https://www.youtube.com/watch?v=1p-DA0zvuQ8

**Pull request merged**